### PR TITLE
Update deployment and caching content and styling

### PR DESCRIPTION
**Pull request by YvonneW**
- **About this release**: Update Adaptive authentication function library info and added FIDO and TOTP REST API doc links.
- **Deployment Patterns**: Added extra info on force_local_cache element.
- **Styling**: Fixed the issue that made the HOME button disappear when in RESTful API docs.

### CODE REVIEW

File: en/docs/administer/configuring-cache-layers.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+enable = "true"
+timeout = "900"
+capacity = "5000"
+
+[cache.idp_cache_by_name]
+enable = "true"
+timeout = "900"
+capacity = "5000"
+```
+
```
Comment (NilukaSripalim) [inline en/docs/administer/configuring-cache-layers.md]: add missing cache layers

[cache.service_provider_cache_id]
enable = true
timeout = "900"
capacity = "5000"
[cache.service_provider_cache_inbound_auth]
enable = true
timeout="900"
capacity="5000"
[cache.jwks_cache]
enable=true
timeout="300"
capacity="5000"

File: en/docs/administer/configuring-cache-layers.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
+# Configuring Cache Layers
+
+WSO2 Identity Server has multiple cache layers which are used to improve performance. The following configurations that can be configured in the `deployment.toml` file in the `<IS_HOME>/repository/conf/` directory are used to manage and configure the cache layers.
+
+```toml tab="Sample identity application management cache layer"
+[cache.framework_session_context_cache]
+enable = "true"
```
Comment (NilukaSripalim) [inline en/docs/administer/configuring-cache-layers.md]: Set  all boolean value without quotation marks/speech marks
e.g. enable = true
please do the same in configs  below also

File: en/docs/setup/deployment-patterns.md [hunk 3]
```diff
@@ -288,11 +288,123 @@ Following configurations need to be done to both WSO2 Identity Server nodes in o
...
+        </ul>
+
+    ??? tip "Caching in WSO2 Identity Server"
+        Historically WSO2 Identity Server used distributed caching to utilize the above-mentioned advantages as well as to minimize the coherence problem. However, in newer deployment patterns where the network is not tightly controlled, distributed caching fail in unexpected ways. Hence, we **no longer recommend using distributed caching**. Instead, it is **recommended to have local caches** (if required) and **cache invalidation messages** (if required) by considering the information given below.
+        <ul>
+            <li><b>The ForceLocalCache property</b></br>
+            When Hazelcast clustering is enabled certain caches act as distributed caches. The `force_local_cache` property in the `<IS_HOME>/repository/conf/deployment.toml` directory is there to mark that all the caches should act like local caches even in a clustered setup. (This is by default set to `true`).
+            ```
+            [server]
+            force_local_cache="true"
```
Comment (NilukaSripalim) [inline en/docs/setup/deployment-patterns.md]: It should be, (Set boolean value without quotes)
[server]
force_local_cache = true

File: en/docs/setup/deployment-patterns.md [hunk 4]
```diff
@@ -288,11 +288,123 @@ Following configurations need to be done to both WSO2 Identity Server nodes in o
...
+                                <li>Typically this applies only when you deploy <b>all the nodes in a single server rack having fiber-optic cables</b>. In any other environments, this implementation will cause cache losses. Thus, this implementation is <b>not recommended for general use</b>.
+                            </ul>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+            </li>
+        </ul>
 
 3. Go to the `<IS_HOME>/repository/conf/deployment.toml` file and add the proxy port as `443`. The port 443 is the Load Balancer frontend port.
```
Comment (NilukaSripalim) [inline en/docs/setup/deployment-patterns.md]: to add the proxy port as `443`, the toml configuration is, 
[transport.https]
proxyPort = "443"

File: en/docs/administer/configuring-cache-layers.md (hunk 1 above)
Comment (YvonneW) [reply en/docs/administer/configuring-cache-layers.md]: Fixed

File: en/docs/setup/deployment-patterns.md (hunk 4 above)
Comment (YvonneW) [reply en/docs/setup/deployment-patterns.md]: Fixed

**Pull request merged**